addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -55,7 +57,7 @@ test("can manually unpin a sub-thread", async () => {
     await contains(".o-mail-DiscussContent-threadName", { value: "New Thread" });
     await click("[title='Thread Actions']");
     await click(".o-dropdown-item:contains('Unpin Conversation')");
-    await contains(".o-mail-DiscussSidebar-item", { text: "New Thread", count: 0 });
+    await contains(".o-mail-DiscussSidebarChannel-itemName:text('New Thread')", { count: 0 });
```

Comment:
Test has negative assert in sidebar but no positive assert when in sidebar. Should add it because I see no such assertion in whole `mail` test suite.